### ISSUE TITLE
Flaky E2E: Editor content loads

### ISSUE THREAD

**Issue by worldomonation** [Automattic/wp-calypso#67350]
### Spec file

specs/fse/fse__temp-smoke-test.ts

### TeamCity ID

8529177

### Logs

```
locator.waitFor: Timeout 90000ms exceeded.
=========================== logs ===========================
waiting for selector ".calypsoify.is-iframe iframe.is-loaded >> control=enter-frame >> iframe[name="editor-canvas"] >> control=enter-frame >> text=Home" to be visible
  waiting for frame ".calypsoify.is-iframe iframe.is-loaded"
============================================================
at Object.<anonymous> (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/test/e2e/specs/fse/fse__temp-smoke-test.ts:53:17)
    at Promise.then.completed (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-circus/build/utils.js:391:28)
    at callAsyncCircusFn (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-circus/build/utils.js:316:10)
    at _callCircusTest (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-circus/build/run.js:218:40)
    at _runTest (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-circus/build/run.js:155:3)
    at _runTestsForDescribeBlock (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-circus/build/run.js:66:9)
    at _runTestsForDescribeBlock (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-circus/build/run.js:60:9)
    at run (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-circus/build/run.js:25:3)
    at runAndTransformResultsToJestFormat (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-circus/build/legacy-code-todo-rewrite/jestAdapterInit.js:170:21)
    at jestAdapter (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-circus/build/legacy-code-todo-rewrite/jestAdapter.js:82:19)
    at runTestInternal (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-runner/build/runTest.js:389:16)
    at runTest (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-runner/build/runTest.js:475:34)
    at Object.worker (/home/teamcity-3/buildAgent/work/c4a9d5b38c1dacde/node_modules/jest-runner/build/testWorker.js:133:12)
```

**Comment by worldomonation** [comment]
This test is particularly egregious. Just take a look:
![image](https://user-images.githubusercontent.com/6549265/188246190-8f8e6d80-f3d7-4c29-a100-ebef7e5ee155.png)

**Comment by worldomonation** [comment]
![image](https://user-images.githubusercontent.com/6549265/188752048-b0b134e8-3f9f-459b-89a0-a07056e54cb3.png)

**Comment by worldomonation** [comment]
According to [this thread](p1662503702003269-slack-CHN6J22MP) the screenshot above is an expected outcome, so perhaps it's better for me to adjust the expectation of the test.

**Comment by worldomonation** [comment]
After the fix at #67514, the failure graph is as follows:

![image](https://user-images.githubusercontent.com/6549265/189702900-c9b53d29-8059-4467-bed3-67a76ba079e5.png)

The chart here starts at September 6 13:36 UTC, which is around the time when the "fix" was merged. Looks like the failure rate has decreased slightly (by about 1%), but not enough to make a difference.
